ENIGMAsystem/SHELL/Graphics_Systems/General/GSbackground.cpp

```diff
@@ -18,13 +18,17 @@
 #include <cstddef>
 
 #include <math.h>
+#include "Graphics_Systems/General/GScolors.h"
 #include "Graphics_Systems/General/GSprimitives.h"
 #include "Graphics_Systems/General/GSbackground.h"
 #include "Graphics_Systems/General/GStextures.h"
 #include "Universal_System/nlpo2.h"
 #include "Universal_System/backgroundstruct.h"
 #include "Universal_System/spritestruct.h"
 
+//Note that this clamps between 0 and 1, not 0 and 255
+#define bind_alpha(alpha) (alpha <= 0 ? 0: alpha >= 1? 1: alpha)
```

Comment:
Nit: In graphics programming, `bind` has a very special meaning; I'd use `clamp`.